StPetersburg/IVEf55.xml

```diff
@@ -0,0 +1,265 @@
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="IVEf55" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Liturgical Book</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0301IV"/>
+                        <idno>IV Ef. 55</idno>
+                        <altIdentifier>
+                            <idno>Platonov cat. II.19</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                          <locus from="3r" to="72v"></locus>
+                          <title type="complete"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <incipit xml:lang="gez">እዝል፡ መስተጋብዕ፡ ዘሠኑይ፡ ወልቡ፡ </incipit>
+                            <note>With musical notation</note>
+
+                              <msItem xml:id="ms_i1.1">
+                                <locus from="3v" to="4r"></locus>
+                                <title type="complete" ref="LIT4518Hymn"></title>
+                                <textLang mainLang="gez"></textLang>
+                                <note>With musical notation</note>
```

Comment:
there is the tag notatedMusic in TEI, could be used as `<notatedMusic><desc>Musical notation present</desc></notatedMusic>`